Kwenta/kwenta
# Fixed the unclickable issue of market dropdown on firefox

## Description
The market dropdown row is unclickable, and the position is off.

![image](https://user-images.githubusercontent.com/4819006/230466666-47497bef-6a7a-46b5-a1c1-4514950b7a85.png)

## Related issue
#2169 

## Motivation and Context


## How Has This Been Tested?
1. Open the market page on both firefox and chrome.
2. Check the market dropdown is in the right place
2. Click the rows of market dropdown to see if the market is open correctly

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/230466226-f70fc504-01f1-46fc-9733-463972f95471.png)
